DESCRIPTION

```diff
@@ -6,15 +6,18 @@ Date: 2015-02-09
 Author: Dirk Eddelbuettel
 Maintainer: Dirk Eddelbuettel <edd@debian.org>
 Description: An R interface to the Pushbullet messaging service which
- provides fast and efficient notifications (and file transfer) between
- computers, phones and tablets.  An account has to be registered at the site
- http://www.pushbullet.com site to obtain a (free) API key.
-Imports: utils, RJSONIO
+    provides fast and efficient notifications (and file transfer) between
+    computers, phones and tablets.  An account has to be registered at the site
+    http://www.pushbullet.com site to obtain a (free) API key.
+Imports:
+    utils,
+    RJSONIO
```

Comment:
All changes in this file are needless changes from stooopid roxygen.  We only need the last line, as I don't like the enforced indentation change.  (But I am just grumbling, this is not really important.)